lego/apps/events/serializers/events.py

```diff
@@ -235,6 +238,17 @@ def get_pending_registration(self, obj):
         except ObjectDoesNotExist:
             return None
 
+    def get_photo_consents(self, obj):
+        request = self.context.get("request", None)
+        if not request or not request.user.is_authenticated:
+            return []
+
+        if not obj.use_consent or len(obj.get_possible_pools(request.user)) == 0:
+            return []
```

Comment:
Why is this not a single if statement? IMO looks better to have everything in one.

Reply:
I split it up both to make it cleaner and to separate logic.

So the first one checks if the user is defined and the second checks if the user should have consents for the event.

I can merge them but I just find short conditional statements alot easier to read and understand.

Reply:
I think neither solution is very elegant. I guess a comment of what you're validating here (like you're stating in the PR) would help.

Reply:
I thought about adding a comment here, but it's very well explained in the commit message.

Reply:
I know, but I'd prefer to not have to git blame something to understand ...

Reply:
I'll add the comment.

But I will say that avoiding `git blame` is a really bad idea. It's super useful to get the full context of a change, with the authors intention and comment on the entire change and not just a single block of code.

Reply:
Yes, I think combining code as docs and a good commit message is the optimal solution :)